src/modules/tas_studio/editor/mod.rs

```diff
@@ -2841,14 +2841,23 @@ impl Editor {
 
         let mut line = "target_yaw_override".to_string();
         // Skip the first frame because it is the initial frame before the start of the TAS.
-        for yaw in &smoothed[1..] {
+        for yaw in &smoothed.1[1..] {
             let yaw = yaw.to_degrees();
             write!(&mut line, " {yaw}").unwrap();
         }
 
         let op = Operation::Insert { line_idx: 0, line };
         self.apply_operation(op)?;
 
+        let mut line = "pitch_override".to_string();
```

Comment:
If the pitch was constant then it probably doesn't need to be smoothed? Also do you generally want to smooth both at once?

Reply:
I am not sure how I would change existing functions to stop working for stream of constant pitch. It just seems a lot easier to mirror it and just works. I see what you mean about second point where someone might just want to smooth the yaw. However, they can delete it easily. Also, adding another option to smooth pitch might be inconvenient as if soneone wants smoothing, just smooth it.